src/Webserver.cc

```diff
@@ -2,8 +2,6 @@
 // http://www.boost.org/doc/libs/1_55_0/doc/html/boost_asio/example/cpp11/echo/blocking_tcp_echo_server.cpp
 
 #include "request_handler.h"
-#include "echo_handler.h"
```

Comment:
Oh if it works without the includes, then we can just leave it out I guess.

Ok. I will fix the empty file name case too